frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -201,7 +203,21 @@ export const HorizontalNav: React.FC<HorizontalNavProps> = React.memo((props) =>
     (extraObjs, key) => ({ ...extraObjs, [key]: _.get(props[key], 'data') }),
     {},
   );
-  const pages = props.pages || props.pagesFor(_.get(props.obj, 'data'));
+  const navTabExtensions = _.get(props.obj, 'data')
+    ? useExtensions<HorizontalNavTab>(isHorizontalNavTab).filter(
```

Comment:
Hooks shouldn't be used conditionally. Probably no harm if it's the only one in the function though.

Reply:
You can call the `useExtensions` hook multiple times.

It's better if a hook is called consistently (i.e. always once, instead of maybe once, per component render).